deploy/charts/emqx-ee/values.yaml

```diff
@@ -0,0 +1,214 @@
+## Default values for emqx.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## It is recommended to have odd number of nodes in a cluster, otherwise the emqx cluster cannot be automatically healed in case of net-split.
+replicaCount: 3
+image:
+  repository: emqx/emqx-enterprise
```

Comment:
@zmstone For EMQX Enterprise 5.0, the docker repository is `emqx/emqx-ee` or `emqx/emqx-enterprise` ?